### PR TITLE
faq: Add questions about supported hardware and OS

### PR DESCRIPTION
**Pull request by philn**
----

Site preview: https://igalia.github.io/wpewebkit.org/faq-hw-os/

### CODE REVIEW

**Review by aperezdc — APPROVED** [review]

**Comment by aperezdc** [comment]
I'll wait a few hours before merging this in case someone else would like to check it as well, maybe @bkardell and/or @meyerweb :mag:

File: about/faq.md [hunk 1]
```diff
@@ -28,6 +28,19 @@ hosting under the webkit.org umbrella. This version of WPE its [released every 6
 months](/release/schedule) from the code hosted on the [upstream repository](https://webkit.org).
 This flavor of WPE is maintained upstream and provides regular [security updates](/security).
 
+## Is WPE supported on any specific hardware System-on-Chip?
+
+WPE has been [ported to a wide range of hardware
+platforms](https://wpewebkit.org/about/supported-hardware.html). The team aims
+to expand the list even further, don't hesitate to contact us if you can't find
+your favorite SoC in the list.
```
[the remark this answers is not in the document]
Comment (philn) [reply about/faq.md:36]: Your suggestion makes sense I think.

[the remark this answers is not in the document]
Comment (meyerweb) [reply about/faq.md:36]: I agree we can merge as-is.  We can add the call for PRs to add hardware later, when we decide on a preferred contact route for people who aren’t interested in/comfortable with submitting PRs.

Also, I’d like to take a crack at copy-editing the FAQ and cleaning up the layout of `supported-hardware.html` **in a separate PR**, if that’s okay with all assembled.

[the remark this answers is not in the document]
Comment (philn) [reply about/faq.md:36]: Yeah sure! The initial FAQ has room for improvements of course :)

**Comment by bkardell** [comment]
Seems fine I think...

**Pull request merged**